pkg/target/builtinplugins_test.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package target_test
+
+import (
+	"testing"
+)
+
+func TestBuiltinPlugins(t *testing.T) {
+	th := NewKustTestHarness(t, "/app")
+	th.writeK("/app", `
+secretGenerator:
+- name: bob
+  kvSources:
+  - pluginType: builtin
+    name: literals
+    args:
+    - FRUIT=apple
+    - VEGETABLE=carrot
+  - pluginType: builtin
+    name: files
+    args:
+    - foo.env
```

Comment:
Please add
```
 - passphrase=phrase.dat
```
to cover the form where the key isn't the file name, but something else.

This is covered for non-plugin secret generation at
https://github.com/monopole/kustomize/blob/master/pkg/target/configmaps_test.go#L48